### ISSUE TITLE
SyntaxError is thrown when no --breakpoints are given

### ISSUE THREAD

**Issue by mncharity** [oroce/polar-h7-rr#4]
The Example invocation in README.md works for me, but not if the `--breakpoints` argument is removed.

This modified Example (no --breakpoints argument):
```
polar-h7-rr\
  --initial-delay 5\
  --length 50s\
  --output my-data.csv\
  --uuid MYUUID
```
throws this
```
/home/mncharity/pad/2017/polar-heart/polar-h7-rr/node_modules/duration-parser/index.js:1024
      throw peg$buildException(null, peg$maxFailExpected, peg$maxFailPos);
      ^
SyntaxError: Expected "+", "-", "0", float or integer but end of input found.
```
on node v6.7.0, with [npm ls --json](https://github.com/oroce/polar-h7-rr/files/1181635/npm_ls_json.txt).

Fyi.  Thank you for your work.

**Comment by oroce** [comment]
Mh that's strange, I just reinstalled everything and I ran a new measurement using this command:

`polar-h7-rr --initial-delay 5 --breakpoints 30s,40s --length 50s --output my-data.csv --uuid f8ccdd4ee826483493d38f6a4fd9aa57`


Can you retry with this exact commad? ^^


```
{
  "name": "polar-h7",
  "version": "1.0.0",
  "dependencies": {
    "debug": {
      "version": "2.6.8",
      "from": "debug@>=2.2.0 <3.0.0",
      "resolved": "https://registry.npmjs.org/debug/-/debug-2.6.8.tgz",
      "dependencies": {
        "ms": {
          "version": "2.0.0",
          "from": "ms@2.0.0",
          "resolved": "https://registry.npmjs.org/ms/-/ms-2.0.0.tgz"
        }
      }
    },
    "duration-parser": {
      "version": "1.0.2",
      "from": "duration-parser@>=1.0.2 <2.0.0",
      "resolved": "https://registry.npmjs.org/duration-parser/-/duration-parser-1.0.2.tgz"
    },
    "humanize-duration": {
      "version": "3.10.1",
      "from": "humanize-duration@>=3.9.1 <4.0.0",
      "resolved": "https://registry.npmjs.org/humanize-duration/-/humanize-duration-3.10.1.tgz"
    },
    "minimist": {
      "version": "1.2.0",
      "from": "minimist@>=1.2.0 <2.0.0",
      "resolved": "https://registry.npmjs.org/minimist/-/minimist-1.2.0.tgz"
    },
    "noble": {
      "version": "1.8.1",
      "from": "noble@>=1.7.0 <2.0.0",
      "resolved": "https://registry.npmjs.org/noble/-/noble-1.8.1.tgz",
      "dependencies": {
        "bplist-parser": {
          "version": "0.0.6",
          "from": "bplist-parser@0.0.6",
          "resolved": "https://registry.npmjs.org/bplist-parser/-/bplist-parser-0.0.6.tgz"
        },
        "debug": {
          "version": "2.2.0",
          "from": "debug@>=2.2.0 <2.3.0",
          "resolved": "https://registry.npmjs.org/debug/-/debug-2.2.0.tgz",
          "dependencies": {
            "ms": {
              "version": "0.7.1",
              "from": "ms@0.7.1",
              "resolved": "https://registry.npmjs.org/ms/-/ms-0.7.1.tgz"
            }
          }
        },
        "xpc-connection": {
          "version": "0.1.4",
          "from": "xpc-connection@>=0.1.4 <0.2.0",
          "resolved": "https://registry.npmjs.org/xpc-connection/-/xpc-connection-0.1.4.tgz",
          "dependencies": {
            "nan": {
              "version": "2.6.2",
              "from": "nan@>=2.0.5 <3.0.0",
              "resolved": "https://registry.npmjs.org/nan/-/nan-2.6.2.tgz"
            }
          }
        },
        "bluetooth-hci-socket": {}
      }
    },
    "say": {
      "version": "0.10.0",
      "from": "say@>=0.10.0 <0.11.0",
      "resolved": "https://registry.npmjs.org/say/-/say-0.10.0.tgz"
    },
    "through": {
      "version": "2.3.8",
      "from": "through@>=2.3.8 <3.0.0",
      "resolved": "https://registry.npmjs.org/through/-/through-2.3.8.tgz"
    }
  }
}
```

**Comment by mncharity** [comment]
>  I ran a new measurement using this command:
> [...] `--breakpoints 30s,40s` [...]

The observed failure required omitting the `--breakpoints` option.

So to replicate, try instead:
`polar-h7-rr --initial-delay 5                       --length 50s --output my-data.csv --uuid f8ccdd4ee826483493d38f6a4fd9aa57`

**Comment by oroce** [comment]
Oh sorry I misunderstood your issue. I fixed in #6. also I released it as v2.0.1
